packages/storycap/package.json

```diff
@@ -93,8 +93,7 @@
     "yargs": "^17.0.0"
   },
   "peerDependencies": {
-    "@storybook/manager-api": "^7.0.0 || ^8.0.0",
-    "@storybook/preview-api": "^7.0.0 || ^8.0.0"
+    "storybook": "^9.0.0"
```

Comment:
The following changes are described in the [official migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropped-support-for-legacy-packages).

- @storybook/manager-api
- @storybook/preview-api